docs/source/using/search.md

```diff
@@ -109,6 +114,59 @@ run.learning_rate in [0.0001, 0.005]
 | `run_1 <hash=a32c910>` | <table><thead> <tr> <th>learning_rate</th><th>batch_size</th> </tr> </thead>  <tbody> <tr> <th>0.001</th><th>32</th> </tr> </tbody></table> |
 | `run_3 <hash=a32c912>` | <table><thead> <tr> <th>learning_rate</th><th>batch_size</th> </tr> </thead>  <tbody> <tr> <th>0.005</th><th>16</th> </tr> </tbody></table> |
 
+3. Search runs based on metrics' last values.
+   
+`metrics` property takes 2 arguments: metric name and context (name is required, context is empty({}) by default).
+
+
+a. Get runs where `accuracy, {}` last value is greater than 0.25
+```python
+run.metrics['accuracy'].last > 0.25
+```
+
+*Result:*
+
+| Run | Parameters | Metrics | 
+|-----|------------|---------|
+| `run_3 <hash=a32c912>` | <table><thead> <tr> <th>learning_rate</th><th>batch_size</th> </tr> </thead>  <tbody> <tr> <th>0.005</th><th>16</th> </tr> </tbody></table> | <table><thead> <tr><th>loss { "subset":"train" } </th><th> loss { "subset":"test" } </th><th> accuracy {} </th></tr></thead> <tbody><tr><th> 30 </th><th> 30 </th><th> 0.3 </th></tr></tbody> </table>|
+
+b. Get runs where `loss, {'subset': 'train'}` last value is smaller than 25
+
+```python
+run.metrics['accuracy', {'subset': 'train'}].last < 25
+```
+*Result:*
+
+| Run | Parameters | Metrics |
+|-----|------------|---------|
+| `run_1 <hash=a32c910>` | <table><thead> <tr> <th>learning_rate</th><th>batch_size</th> </tr> </thead>  <tbody> <tr> <th>0.001</th><th>32</th> </tr> </tbody></table> | <table><thead> <tr><th>loss { "subset":"train" } </th><th> loss { "subset":"test" } </th></tr></thead> <tbody><tr><th> 10 </th><th> 10 </th><th></tbody> </table>|
+| `run_2 <hash=a32c911>` | <table><thead> <tr> <th>learning_rate</th><th>batch_size</th> </tr> </thead>  <tbody> <tr> <th>0.0007</th><th>64</th> </tr> </tbody></table> | <table><thead> <tr><th>loss { "subset":"train" } </th><th> loss { "subset":"test" } </th><th> accuracy {} </th></tr></thead> <tbody><tr><th> 20 </th><th> 20 </th><th> 0.2 </th></tr></tbody> </table>|
+
+4. Search runs where `duration` is smaller than 10s (to filter out failed runs for example)
+```python
+run.duration < 10
+```
+
+This will yield an empty result on our test runs, but will have a use case on real life experiments to filter the runs that finished under 10 seconds and then archive them as they've probably crashed or were stopped. 
```

Comment:
will it yield an empty result or will all the runs be returned?

Reply:
Yup, well noticed, I initially wrote the text for `run.duration > 10` query, forgot to change it 😀